tests/test_context.py

```diff
@@ -243,42 +246,8 @@ def setUpClass(cls):
         component.registry.register(name="parent_with_args", component=cls.ParentComponentWithArgs)
         component.registry.register(name="variable_display", component=VariableDisplay)
 
-    @override_settings(
-        COMPONENTS={
-            "context_behavior": "django",
-        }
-    )
-    def test_parent_args_can_be_drawn_from_context__django(self):
```

Comment:
Thanks to the `parametrize_context_behavior` decorator, we could remove duplicate tests that differed only in the `context_behavior` mode.